tests/models/layers/test_flash_attn.py

```diff
@@ -362,13 +360,14 @@ def gen_bias():
 
     output_2.sum().backward()
 
-    assert torch.allclose(output_1, output_2)
+    _assert_approx_equal(output_1, output_2)
```

Comment:
note: had to change this because torch attention and flash attention produced values that were more different to each other than triton and flash